docs/decisions/0013-library-reference-content-block.rst

```diff
@@ -0,0 +1,61 @@
+Referencing Content Blocks in Library V2
+--------------------------------------------------
+
+Status
+=======
+Pending
+
+Context
+=======
+Currently, the courses are being stored in the Modulestore and the libraries are stored in both Modulestore and the Blockstore. We want the library content to be stored in the Blockstore.
+We want to have a content reuse mechanism that allows authors to incorporate content (XBlocks) from content libraries into their courses without making a copy of it; in other words, we want a referencing mechanism for library content. 
+
+We are currently building out the functionality of libraries v2, and trying to migrate all usage of content libraries onto Blockstore.
+The benefits of using the Blockstore backed v2 libraries are:
+
+#. Blockstore is much more flexible than modulestore and can store anything that can be represented as a file - unlike modulestore, which only stores courses built of XBlocks. Also, the Blockstore is being moved into the edx-platform.
+#. simpler to maintain than modulestore
+#. increases re-usability of the content
```

Comment:
All these blockstore benefits bullets should be stated consistently, like:
- The blockstore is more flexible...
- The blockstore is simpler to....
- The blockstore increases...

OR:

The blockstore:
- is more flexible...
- is simpler...
- increases...

Reply:
Done, I have chosen the first format